index.html

```diff
@@ -1,80 +1,78 @@
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;600;700&family=Lato:wght@300;400;700&family=Poppins:wght@400;500;600;700&family=Roboto:wght@400;500;700&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="./styles.css">
 </head>
-<body>
-  
-</body>
-</html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>lengushuru Portoflio</title>
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link
-      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;600;700&family=Lato:wght@300;400;700&family=Poppins:wght@400;500;600;700&family=Roboto:wght@400;500;700&display=swap"
-      rel="stylesheet">
-    <link rel="stylesheet" href="./styles.css">
-  </head>
 
-  <body>
-    <header>
-      <nav class="nav-bar">
-        <div class="page--logo">
-          <a class="page--link " href="#">lengushuru</a>
-        </div>
-        <div class="nav--menu" id="nav-toggle">
-          <div></div>
-          <div></div>
-          <div></div>
-        </div>
-        <div class="nav-menu" id="nav-menu">
-          <div class="nav-close" id="nav-close">
-            <i class="bx bx-x"></i>
-          </div>
-          <div class="nav-links">
-            <a class="nav-item" href="#portfolio">Portoflio</a>
-            <a class="nav-item" href="#about-me">About</a>
-            <a class="nav-item" href="#container">Contact</a>
-          </div>
+<body>
+  <header>
+    <nav class="nav-bar">
+      <div class="page--logo">
+        <a class="page--link " href="#">lengushuru</a>
+      </div>
+      <div class="nav--menu" id="nav-toggle">
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+      <div class="nav-menu" id="nav-menu">
+        <div class="nav-close" id="nav-close">
+          <i class="bx bx-x"></i>
         </div>
-        <div class="desk-top">
-          <a class="nav-ite" href="#portfolio">Portoflio</a>
-          <a class="nav-ite" href="#about-me">About</a>
-          <a class="nav-ite" href="#container">Contact</a>
+        <div class="nav-links">
+          <a class="nav-item" href="#portfolio">Portoflio</a>
+          <a class="nav-item" href="#about-me">About</a>
+          <a class="nav-item" href="#container">Contact</a>
+          <a class="nav-item" href="#container"><img src="./images/mail_icon.png" alt=""></a>
         </div>
-      </nav>
-    </header>
-    <section class="hero">
+      </div>
+      <div class="desk-top">
+        <a class="nav-ite" href="#portfolio">Portoflio</a>
+        <a class="nav-ite" href="#about-me">About</a>
+        <a class="nav-ite" href="#container">Contact</a>
```

Comment:
- [ ] Great job on adding the `href` to your links 👏 However they're not functioning correctly because you're not using the id that you put in the `href` inside the sections. For example, if you want a link to take you to the works sections (portfolio) you have to put the `id` of the works section inside the `href` attribute of that link 👍 Kindly fix the issue 💯